dexbot/strategies/staggered_orders.py

```diff
@@ -258,6 +258,8 @@ def allocate_base_asset(self, base_balance, *args, **kwargs):
             else:
                 # Cancel highest buy order
                 self.cancel(self.buy_orders[0])
+                self.place_higher_buy_order(self.buy_orders[0]) # Shouldn't [0] be the new "highest"?
```

Comment:
self.buy_orders does not update here at all so after cancellation [1] is highest since [0] is no more. It depends if you want the higher buy order to be based on the previously highest or the highest at the time.

Reply:
Same applies below on the sell orders.

Reply:
The new highest